documentation/release-notes/13.x.x/13.12.0/README.md

```
@@ -0,0 +1,16 @@
+
+{% hint style="info" %}
+**Release date 21-01-2026**
+{% endhint %}
+
+## New Features
+
+## Enhancements
+
+* **Clarified Semver requirement for case definition versions**
+
+    - The creation modal for case definitions now explicitly states that the version must adhere to Semantic Versioning.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)